build.xml

```diff
@@ -209,7 +209,12 @@
     <property name="reports" value="${build}/reports"/>
 
     <mkdir dir="${build}/ant-elfanalyser" />
-    <javac classpath="lib/ant.jar" destdir="${build}/ant-elfanalyser" includeantruntime="false">
+    <javac classpath="lib/ant.jar" 
+           destdir="${build}/ant-elfanalyser" 
+           includeantruntime="false"
+           source="1.6"
```

Comment:
Good point, I admit I did not think about that. But I don't see ${compatiblity} as the correct version. The analyser class that is generated by this task exists only at build-time and is used to guess the arm variant we are building for. 

The requirement is therefore: The targetversion must be supported by the build environment. 1.6 is still supported by OpenJDK 9 and is older than the oldest available supported JDK I'm aware of (Redhat supports OpenJDK 7 and has dropped support for JDK 6).

So I'd keep it as it is.